### PR TITLE
[AUTO] Update logic of setting TPUT by default for MULTI and AUTO.

### PR DESCRIPTION
**Pull request by yangwang201911**
Signed-off-by: Wang, Yang <yang4.wang@intel.com>

### Details:
 - set TPUT by default for MULTI when user doesn't specify any below properties
```
ov::inference_num_threads
ov::num_streams
ov::affintity
ov::hint::performance_mode
```
- Update logic of the TPUT default setting for AUTO
### Tickets:
 - CVS-87444

### CODE REVIEW

**Comment by wangleis** [comment]
Please double check if we need additional change in benchmark_app when benchmark use MULTI:GPU,CPU with Latency hint.

**Review by wangleis — APPROVED** [review]

File: src/plugins/auto/plugin.cpp [hunk 1]
```diff
@@ -96,6 +96,31 @@ std::vector<DeviceInformation> MultiDeviceInferencePlugin::ParseMetaDevices(cons
     // last token in the string (which has no comma after that)
     devicesWithRequests.push_back(priorities.substr(i, priorities.length() - i));
 
+    auto setTputAsDefault = [&](const std::string& targetDevice,
+                               std::map<std::string, std::string>& deviceConfig,
+                               const std::map<std::string, std::string>& mergedConfig) {
+        if (GetName() == "AUTO" && deviceConfig.find(PluginConfigParams::KEY_PERFORMANCE_HINT) == deviceConfig.end() &&
+            mergedConfig.find(targetDevice) == mergedConfig.end()) {
```
Comment (songbell) [inline src/plugins/auto/plugin.cpp:103]: when will mergedConfig be triggered here?

Comment (yangwang201911) [reply src/plugins/auto/plugin.cpp:103]: you mean when below codes will be executed, right?
`mergedConfig.find(targetDevice) == mergedConfig.end()`

Comment (songbell) [reply src/plugins/auto/plugin.cpp:103]: yeah, what case it is targeting to handle?

Comment (yangwang201911) [reply src/plugins/auto/plugin.cpp:103]: those codes will be triggered to check if any secondary properties setting to target device if all of below conditions are met.
1. plugin name is AUTO.
2. no performance hint setting to target device from user through AUTO plugin.

Comment (songbell) [reply src/plugins/auto/plugin.cpp:103]: can  you help paste some code snippets to explain your comment?

Comment (yangwang201911) [reply src/plugins/auto/plugin.cpp:103]: MULTI set TPUT like AUTO if user doesn't specify performance hint to MULTI or to HW device. @wangleis Is it expected as we discuss?

**Review by wgzintel — APPROVED** [review]

**Pull request merged**